packages/lib-sourcify/src/Verification/VerificationTypes.ts

```diff
@@ -0,0 +1,34 @@
+import { StringMap } from '../Compilation/CompilationTypes';
+import { SourcifyLibError } from '../SourcifyLibError';
+import { Transformation, TransformationValues } from './Transformations';
+
+export interface BytecodeMatchingResult {
+  match: 'perfect' | 'partial' | null;
+  libraryMap?: StringMap;
+  populatedRecompiledBytecode: string;
+  transformations: Transformation[];
+  transformationValues: TransformationValues;
+  message?: string;
+}
+
+export type VerificationErrorCode =
+  | 'CANT_FETCH_BYTECODE'
+  | 'CONTRACT_NOT_DEPLOYED'
+  | 'COMPILED_BYTECODE_IS_ZERO'
+  | 'EXTRA_FILE_INPUT_BUG'
+  | 'CREATION_BYTECODE_MATCH_ERROR'
+  | 'NO_MATCH'
+  | 'ONCHAIN_RUNTIME_BYTECODE_NOT_AVAILABLE'
+  | 'ONCHAIN_CREATION_BYTECODE_NOT_AVAILABLE'
+  | 'BYTECODE_LENGTH_MISMATCH';
```

Comment:
Can we still rename the error codes to use lower snake_case? That way they would match the format of API v2 errors and we could directly use them.